modules/ROOT/pages/_partials/commons/common-scopes-collections-manage.adoc

```diff
@@ -61,6 +61,8 @@ NOTE: Scope and collection names are case sensitive.
 include::{root-partials}block_tabbed_code_example.adoc[]
 :param-tags!:
 
+NOTE: You cannot create an empty user-defined scope. 
+Scopes are an entity that contains collections, they are implicity created when a collection with documents associated to the provided scope is created.
```

Comment:
In general, I'm having trouble understanding this sentence. Seems to be saying that Scopes are implicity[sic?] created when a collection is associated to Scope? I'm not sure how it would create a scope when it's already associated with one, but it might be a misunderstanding or lack of context on my part.

Reply:
Yeah I probably need to get some clarification on this, from what I understand this is basically the case but the order seems a little backwards to me.